tests/fixtures/modified_survey_point.json

```diff
@@ -0,0 +1,251 @@
+                    "coordinates": [
+                        89,
+                        89
+                    ]
+                }
+            },
+            "oldVersion": {
+                "type": "Feature",
+                "properties": {
+                    "man_made": "survey_point",
+                    "osm:version": 8,
+                    "name": "Another survey point",
+                    "source": "IBGE"
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [
+                        89,
+                        89
+                    ]
+                }
+            },
+            "expectedResult": false
+        },
+        {
+            "description": "Flag if survey_point lost its main tag",
+            "newVersion": {
+                "type": "Feature",
+                "properties": {
+                    "historic": "monument",
+                    "osm:version": 9
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [
+                        89,
+                        89
+                    ]
+                }
+            },
+            "oldVersion": {
+                "type": "Feature",
+                "properties": {
+                    "man_made": "survey_point",
+                    "osm:version": 8,
+                    "name": "Another survey point",
+                    "source": "IBGE"
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [
+                        89,
+                        89
+                    ]
+                }
+            },
+            "expectedResult": {
+                "result:modified_survey_point": true
+            }
+        },
+        {
+            "description": "Flag if survey_point lost its main tag",
+            "newVersion": {
+                "type": "Feature",
+                "properties": {
+                    "man_made": "obelisk",
+                    "osm:version": 9
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [
+                        89,
+                        89
+                    ]
+                }
+            },
+            "oldVersion": {
+                "type": "Feature",
+                "properties": {
+                    "man_made": "survey_point",
+                    "osm:version": 8,
+                    "name": "Another survey point",
+                    "source": "IBGE"
+                },
+                "geometry": {
+                    "type": "Point",
+                    "coordinates": [
+                        89,
+                        89
+                    ]
+                }
+            },
+            "expectedResult": {
+                "result:modified_survey_point": true
+            }
+        }
+    ]
```

Comment:
The last test case seems like same as the test case just before it

Reply:
In one case the tag `man_made=survey_point` was changed to `historic=monument`. In the other, it changed to `man_made=obelisk`